teamcity-server/src/main/java/com/cyberark/server/ConjurBuildStartContextProcessor.java

```diff
@@ -5,38 +5,47 @@
 import jetbrains.buildServer.BuildProblemData;
 import jetbrains.buildServer.serverSide.*;
 import jetbrains.buildServer.serverSide.oauth.OAuthConstants;
+import jetbrains.buildServer.serverSide.parameters.types.PasswordsProvider;
 
+import java.util.Collection;
 import java.util.Map;
 import java.util.List;
 import java.util.ArrayList;
 
 import com.cyberark.common.*;
 
-public class ConjurBuildStartContextProcessor implements BuildStartContextProcessor {
+public class ConjurBuildStartContextProcessor implements BuildStartContextProcessor, PasswordsProvider {
 
-    // This method will return one SProjectFeatureDescriptor that represents the CyberArk Conjur Connection
-    //   provided in the project Connections. This method will return null if no Connection can be found and will throw
-    //   a MultipleConnectionsReturnedException if more than one connection was found.
-    private SProjectFeatureDescriptor getConnectionType(SProject project, String providerType) throws MultipleConnectionsReturnedException {
-        List<SProjectFeatureDescriptor> connections = new ArrayList<SProjectFeatureDescriptor>();
+    // This method will params that represents the CyberArk Conjur Connection
```

Comment:
Looks like something is missing here, maybe just the word "return"?

Reply:
Thank you - fixed.